web.browser/src/containers/PostList/PostList.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import PostToolbar from '../../components/PostToolbar';
+import Post from '../../components/Post';
+
+
+const PostList = ({ posts, updateVote, sortNewest, sortPopular }) => {
+  return (
+    <div>
+      <div style={{ marginLeft: '256px' }}>
```

Comment:
Interesting 2^8 margin.

If you're going to use the `styles` import elsewhere, you should probably use it everywhere.